CMakeLists.txt

```diff
@@ -128,8 +131,15 @@ function(ADD_PARQUET_TEST REL_TEST_NAME)
     set(TEST_PATH ${CMAKE_CURRENT_SOURCE_DIR}/${REL_TEST_NAME})
   endif()
 
-  add_test(${TEST_NAME}
-    ${BUILD_SUPPORT_DIR}/run-test.sh ${TEST_PATH})
+  if (PARQUET_TEST_MEMCHECK)
+	SET_TARGET_PROPERTIES(${TEST_NAME}
```

Comment:
This does not work on Linux as it will override all previously set target properties and thus compiling will fail due to the wrong `tr1/tuple` handling. The following line worked for me and seems to be the common way to append to target properties (note the explicit space between `"` and `-D`):

```
SET_PROPERTY(TARGET ${TEST_NAME} APPEND_STRING PROPERTY COMPILE_FLAGS " -DPARQUET_VALGRIND")
```